fairlearn/reductions/moments/conditional_opportunity.py

```diff
@@ -56,6 +65,18 @@ class DemographicParity(ConditionalOpportunity):
     """
     short_name = "DemographicParity"
 
+    def __init__(self, from_X=True, column_name=None, column_index=None):
+        super().__init__()
+        self.from_X = from_X
+        if column_name is None and column_index is None:
+            # raise ValueError("Did not specify column_name or column_index")
```

Comment:
Need to figure out why these raise clauses were giving VS Code grief. For now, I'm skipping the error check

Reply:
What grief?

Reply:
As I recall flake8 was raising issues because it thought the exceptions would always be thrown. Basically, something I didn't want to deal with while trying to get a happy path working. I may have been overly paranoid about what it was saying.